developer/openshift/reset.sh

```diff
@@ -215,10 +259,47 @@ uninstall_operators(){
       kubectl delete ns "$tkn_results_ns"
     fi
 
+    # Checks if the operators are uninstalled successfully
+    mapfile -t operators < <(kubectl get operators | grep -iE "openshift-gitops-operator" | cut -d " " -f 1)
+    if (( ${#operators[@]} >= 1 )); then
+        printf "\n[ERROR] Couldn't uninstall giops operator, please try removing it manually." >&2
+        exit 1
+    fi
+
+    # Checks if the Tekton controllers are uninstalled successfully
+    mapfile -t controllers < <(kubectl get ns | grep -iE "tekton-results|tekton-chains|pipelines-as-code" | cut -d " " -f 1)
+    if (( ${#controllers[@]} >= 1 )); then
+        printf "\n[ERROR] Couldn't remove Tekton controllers, please try removing them manually." >&2
+        exit 1
+    fi
+
+    printf "\nGitops operator and Tekton controllers are successfully uninstalled.\n"
+}
+
+is_hard_reset_enabled() {
```

Comment:
Instead of that function, I would remove line 30, add `RESET_HARD=${RESET_HARD:-}` in the init, and just use `[ -n "$RESET_HARD" ]` everywhere.